torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -1179,6 +1195,45 @@ def call_triton_kernel(
                 "Please use a Config in @triton.autotune instead."
             )
 
+        # We support running a single Autotuner for each Triton kernel
+        # Currently, if there are multiple autotuning decorators, the subsequent ones will be silently ignored
+        # We raise an error here to avoid silent incorrectness
+        iter_kernel = variable.kernel
+        autotuner_count = 0
+        while not isinstance(iter_kernel, JITFunction):
+            if isinstance(iter_kernel, Autotuner):
+                autotuner_count += 1
+            if autotuner_count > 1:
+                self.raise_unsupported(
+                    "Passing multiple @triton.autotune decorators is not supported. "
+                    "Please use a single @triton.autotune decorator instead."
+                )
+            iter_kernel = iter_kernel.fn
```

Comment:
Wondering if we could move erroring on nested autotuners to a separate PR, as it's not strictly required for this one?

Reply:
this is needed for the heuristics PR, but I can put this in a separate PR